samples/matter/common/src/board/board.cpp

```diff
@@ -18,14 +18,17 @@
 
 LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
 
-Board Board::sInstance;
+Nordic::Matter::Board Nordic::Matter::Board::sInstance;
+
+namespace Nordic {
+namespace Matter {
 
 bool Board::Init(button_handler_t buttonHandler, LedStateHandler ledStateHandler)
 {
 #ifdef CONFIG_DK_LIBRARY
 	/* Initialize LEDs */
-	LEDWidget::InitGpio();
-	LEDWidget::SetStateUpdateCallback(LEDStateUpdateHandler);
+	Nordic::LEDWidget::InitGpio();
```

Comment:
This definition is already in the Nordic namespace so you don't need such changes.

Reply:
Yes, you are right, the reason is I modified files one by one and fixed compilation errors gradually, so there are some unnecessary Nordic oraz Nordic::Matter usages in a modules that also use them. I will go through all files once again and remove such cases.

Reply:
@kkasperczyk-no I hope you leaked "oraz" here on purpose because it sounds great to me :)

Reply:
Wtf :smile: I have no idea how it happened. Maybe I was thinking about too many things at the same time.